core/src/test/java/nu/yona/server/goals/service/ActivityCategoryServiceIntegrationTest.java

```diff
@@ -30,11 +29,11 @@
 
 import nu.yona.server.goals.entities.ActivityCategory;
 import nu.yona.server.goals.entities.ActivityCategoryRepository;
-import nu.yona.server.goals.service.ActivityCategoryServiceTests.TestConfiguration;
+import nu.yona.server.goals.service.ActivityCategoryServiceIntegrationTest.TestConfiguration;
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(classes = { TestConfiguration.class })
-public class ActivityCategoryServiceTests
+public class ActivityCategoryServiceIntegrationTest
 {
 	private Set<ActivityCategory> activityCategories = new HashSet<ActivityCategory>();
```

Comment:
Yes it does, because I am returning a mocked `ActivityCategoryRepository` repository bean in the `TestConfiguration` class below instead of a real instance.
Maybe there is some way to return a real instance that works nicely, but the example I found [http://stackoverflow.com/questions/27856266/how-to-make-instance-of-crudrepository-interface-during-testing-in-spring] is far more complicated.